strawberry/types/type_resolver.py

```diff
@@ -138,6 +139,19 @@ class if one is not set by either using an explicit strawberry.field(name=...) o
             origin = origins.get(field.name, cls)
             module = sys.modules[origin.__module__]
 
+            if isinstance(field_type, TypeVar):
+                specialized_type_var_map = get_specialized_type_var_map(cls)
+                # If field_type is specialized and a TypeVar, replace it with its
```

Comment:
`replace it with its` I think a word is missing here :D

Reply:
lol